node/src/command.rs

```diff
@@ -141,7 +141,7 @@ impl SubstrateCli for Cli {
 
     fn load_spec(&self, id: &str) -> Result<Box<dyn sc_service::ChainSpec>, String> {
         Ok(match id {
-            "release" => Box::new(chain_spec::release_config(fetch_keys(TEST_KEYS_URL)?)?),
+            "testnet" => Box::new(chain_spec::testnet_config(fetch_keys(TEST_KEYS_URL)?)?),
```

Comment:
Should we make testnet the default by matching `""` here?

Reply:
Good question... off the top of my head, I want people to use `--testnet`

Reply:
OK, I am fine with that